lib/nmatrix/yale_functions.rb

```diff
@@ -89,7 +93,8 @@ def yale_ja_d_keys_at i
   # call-seq:
   #     yale_ja_d_keys_set_at(i) -> Set
   #
-  # Returns the diagonal and non-diagonal column indices stored in a given row.
+  # Returns the diagonal and non-diagonal column
+  # indices stored in a given row.
```

Comment:
How about we indent multi-line comments with two spaces? That makes it obvious that the next line is a continuation of the previous.